src/model_io/urdf/CMakeLists.txt

```diff
@@ -50,6 +50,9 @@ include_directories(${IDYNTREE_TREE_INCLUDE_DIRS})
 include_directories(SYSTEM ${urdfdom_headers_INCLUDE_DIRS}
                            ${urdfdom_INCLUDE_DIRS})
 
+if(IDYNTREE_COMPILE_TESTS)
+    add_subdirectory(tests)
+endif()
```

Comment:
If you move the `add_subdirectory(tests)` here, how can the test link to the `idyntree-modelio-urdf`that still needs to get added?